crates/kitsune_p2p/dht_arc/src/test/arc_stability.rs

```diff
@@ -18,18 +18,27 @@ fn full_len() -> f64 {
     2f64.powi(32)
 }
 
+#[test]
+fn min_redundancy_is_maintained() {
+    todo!("Check that min redundancy is maintained at all times");
+}
+
 #[test]
 fn parameterized_stability_test() {
     let n = 500;
     let j = 1f64 / n as f64 / 3.0;
 
-    let kind = PeerViewParams::Alpha;
+    let strat = PeerStratAlpha {
+        // redundancy_target: 100,
+        ..Default::default()
+    };
+    let kind = PeerStrat::Alpha(strat);
 
     let peers = simple_parameterized_generator(n, j, ArcLenStrategy::Constant(0.1));
     report(peers, kind);
 }
 
-fn report(mut peers: Vec<DhtArc>, kind: PeerViewParams) {
+fn report(mut peers: Vec<DhtArc>, kind: PeerStrat) {
```

Comment:
I'm not sure how "Strat" adds clarity over "ViewParams"... but don't want to bikeshed.